presto-jmx/src/main/java/com/facebook/presto/connector/jmx/JmxMetadata.java

```diff
@@ -90,14 +128,17 @@ public JmxTableHandle getTableHandle(ConnectorSession session, SchemaTableName t
             MBeanInfo mbeanInfo = mbeanServer.getMBeanInfo(objectName.get());
 
             ImmutableList.Builder<JmxColumnHandle> columns = ImmutableList.builder();
-            columns.add(new JmxColumnHandle(connectorId, "node", createUnboundedVarcharType()));
-            for (MBeanAttributeInfo attribute : mbeanInfo.getAttributes()) {
-                if (!attribute.isReadable()) {
-                    continue;
-                }
-                columns.add(new JmxColumnHandle(connectorId, attribute.getName(), getColumnType(attribute)));
-            }
-            return new JmxTableHandle(connectorId, objectName.get().toString(), columns.build());
+            columns.add(new JmxColumnHandle(connectorId, NODE_COLUMN_NAME, createUnboundedVarcharType()));
+
+            // Since this method is being called on all nodes in the cluster, we must ensure (by sorting)
+            // that attributes are in the same order on all of them.
+            Arrays.stream(mbeanInfo.getAttributes())
```

Comment:
This seems like a fix for a pre-existing bug. Can you split it into a separate commit?